app/jobs/risc_delivery_job.rb

```diff
@@ -100,4 +100,8 @@ def faraday
   def inline?
     queue_adapter.is_a?(ActiveJob::QueueAdapters::InlineAdapter)
   end
+
+  def self.warning_messages
+    NETWORK_ERRORS + [RedisRateLimiter::LimitError]
+  end
```

Comment:
I find the name of this method a little misleading, these are error classes, not messages, WDYT about a name like `.warning_error_classes`
```suggestion
  def self.warning_error_classes
    NETWORK_ERRORS + [RedisRateLimiter::LimitError]
  end
```

Reply:
We could also define this on `ApplicationJob` so that we can avoid the `try` in the job log subscriber. Is it worth doing that?

Reply:
I moved it to ApplicationJob. Good idea.